BootloaderCommonPkg/Include/Library/FirmwareUpdateLib.h

```diff
@@ -503,4 +505,41 @@ EFIAPI
 ClearFwUpdateTrigger (
   VOID
   );
+
+
+/**
+  Main routine for Command updates.
+
+  This function has the logic to perform CMD  update.
+
+  @param[in] CapImageHdr    The pointer to the firmware update capsule image.
+
+  @retval  EFI_SUCCESS      Update successful
+  @retval  other            error status from the update routine
+**/
+EFI_STATUS
+FwCmdUpdateProcess (
+  EFI_FW_MGMT_CAP_IMAGE_HEADER  *CapImageHdr
+  );

```

Comment:
Library/FirmwareUpdateLib.h is the interfaces file provided by FirmwareUpdateLib.
FwCmdUpdateProcess () is not part of FirmwareUpdateLib, should not be added to this header file.
it could be added to a internal header file in Firmware update driver (FirmwareUpdateHelper.h?)

Reply:
Done. Moved FwCmdUpdateProcess() to local header FirmwareUpdateHelper.h.